src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -73,6 +76,12 @@ public void writePrimitiveArrayType() {
 
   /** =============================== EXPRESSIONS =============================== */
   @Test
+  public void writeStringObjectValue() {
+    StringObjectValue s = StringObjectValue.builder().setValue("test").build();
+    assertThat(s.value()).isEqualTo("test");
```

Comment:
When writing this in Java source, this should be `"\"test\""`, otherwise it will show up as `String x = test`.

Reply:
done.